src/main/java/Parser.java

```diff
@@ -0,0 +1,103 @@
+import java.util.ArrayList;
+
+public class Parser {
+
+    public static void parse(String input, TaskList taskList) {
+        String[] array = input.split(" ");
+        String execute = array[0];
+        switch(execute) {
+            case("list"):

```

Comment:
The switch statement should have the following form. There is no indentation for case clauses.